python/pyspark/shell.py

```diff
@@ -31,6 +31,7 @@
 from pyspark import SparkConf
 from pyspark.context import SparkContext
 from pyspark.sql import SparkSession, SQLContext
+from pyspark.sql.functions import C, col
```

Comment:
This is worth calling out as a significant change, as it means C and col would automatically be imported into the REPL.  Makes sense to do this because col is by far the most commonly used function in all of pyspark and it increases user-friendliness.